### PR TITLE
#1085 Align tool names in scripts

### PR DESCRIPTION
**Pull request by CREITZ25**
#1085 Align tool names in scripts

### CODE REVIEW

**Comment by CREITZ25** [comment]
@hohwille
In order to adjust the cobigen commandlet according to the specifications, the path cobigen-cli must be changed to cobigen. The variable COBIGEN_CLI_HOME should also be adjusted to COBIGEN_HOME. However, the problem arises that the variable is already used in the cobigen environment. Therefore, in the scripts/devon.properties file, the variable COBIGEN_HOME is already set to a configuration file or directory. How do I have to proceed here?

https://github.com/devonfw/cobigen/blob/master/documentation/cobigen-core_configuration.asciidoc

**Comment by CREITZ25** [comment]
@hohwille after the issue has been implemented, the doGetToolFolderName function is free of the exceptions and generates the name of the software from the transferred software path. Perhaps the function should be renamed to doGetToolName, or replace the calls directly with the contents of the function.

**Comment by CREITZ25** [comment]
node is already adapted

**Comment by hohwille** [comment]
> @hohwille after the issue has been implemented, the doGetToolFolderName function is free of the exceptions and generates the name of the software from the transferred software path. Perhaps the function should be renamed to doGetToolName, or replace the calls directly with the contents of the function.

yes, we should remove this function as it is more or less nothing than `basename`.
so you can change things like
```
doListSoftwareVersions "$(doGetToolFolderName "${0}")"
```
to simply
```
doListSoftwareVersions "${0}"
```

Theoretically in some cases also a path could be provided as argument where we would need to do `${0/*\//}` but I can not find any place where the function is used with a path rather than just the tool name itself.

**Review by hohwille — CHANGES_REQUESTED** [review]
@CREITZ25 thanks for implementing this story. Looks good and you have catched all places to adapt except the very last one I commented. 👍 
IMHO it is fine to leave `MAVEN_HOME` and not rename it to `MVN_HOME` and we already discussed the name clash between `COBIGEN_HOME` so also `COBIGEN_CLI_HOME` must stay.

As we have some dependencies with other stories, those have to be implemented and merged, before we can merge this one as otherwise it will break existing functionality on master and make tests red...

File: scripts/src/main/resources/scripts/command/mvn
Comment (CREITZ25) [reply scripts/src/main/resources/scripts/command/mvn]: Of course, I forgot it

**Review by hohwille — APPROVED** [review]
@CREITZ25 thanks for lightning fast update. Now ready but waiting for dependent issues to be merged first (see #1085 where I added the dependencies to the comment).

**Review by hohwille — APPROVED** [review]
@CREITZ25 IMHO all is correct here and this could be merged.
Can we align on the different related stories that we all need to merge together to result in a working state as just merging parts of the epic like only this PR may result in a broken state...

**Pull request merged**
